FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentRecipeMaterialList.cs

```diff
@@ -0,0 +1,32 @@
+using FFXIVClientStructs.FFXIV.Client.System.String;
+using FFXIVClientStructs.FFXIV.Component.GUI;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Agent;
+
+[Agent(AgentId.RecipeMaterialList)]
+[VTableAddress("48 8D 05 ?? ?? ?? ?? 48 C7 43 ?? ?? ?? ?? ?? 48 89 03 33 C0 89 43 28", 3)]
+[StructLayout(LayoutKind.Explicit, Size = 0x40)]
+public unsafe partial struct AgentRecipeMaterialList {
+    [FieldOffset(0)] public AgentInterface AgentInterface;
+    [FieldOffset(0x28)] public ushort RecipeId;
+
+    [FieldOffset(0x2C)] public uint Amount;
+
+    [FieldOffset(0x34)] public bool WindowLocked;
+    [FieldOffset(0x38)] public RecipeData* Recipe;
+
+    [MemberFunction("E8 ?? ?? ?? ?? EB B1 48 8B 4B 28")]
+    public readonly partial void OpenByRecipeId(uint recipeId, uint amount = 1);
```

Comment:
`recipeId` should be a `ushort` no?
![word ptr](https://screenshots.wildwolf.dev/ida64_a75ba7783e.png)
![unsigned __int16](https://screenshots.wildwolf.dev/ida64_4db4de5d66.png)

Reply:
Ah yes. I forgot.